streams/src/test/java/org/apache/kafka/streams/processor/TopologyBuilderTest.java

```diff
@@ -586,7 +586,8 @@ public void shouldAddInternalTopicConfigForRepartitionTopics() {
         final TopicsInfo topicsInfo = builder.topicGroups().values().iterator().next();
         final InternalTopicConfig topicConfig = topicsInfo.repartitionSourceTopics.get("appId-foo");
         final Map<String, String> properties = topicConfig.getProperties(Collections.<String, String>emptyMap(), 10000);
-        assertEquals(4, properties.size());
+        assertEquals(5, properties.size());
+        assertEquals(properties.get(TopicConfig.RETENTION_MS_CONFIG), String.valueOf(Long.MAX_VALUE));
```

Comment:
Nit: `assertEquals` has "expected value" as first parameter (some for other test).